sources/reference/usage/report.md

```diff
@@ -39,11 +46,11 @@ You can export various formats of report by specifying what you want using the
 For example, to generate a PDF report you can run the following command:
 
 ```
-$ chaos report --export-format=pdf chaos-report.json report.pdf
+chaos report --export-format=pdf chaos-report.json report.pdf
 ```
 
 An HTML report can be produced using:
 
 ```
-$ chaos report --export-format=html5 chaos-report.json report.html
+chaos report --export-format=html5 chaos-report.json report.html
```

Comment:
why remove the $ sign here?

useful for command lines.

Reply:
I've removed them in all the other changes - if you copy and pasted that into your command line using the button, it will fail because it's prepended with `$`

Reply:
I see. Okay let's remove them for now. Marc implemented it nicely so that the $ is not copied on reliably :p

Reply:
@Lawouach yeah the reliably docs do it really nicely 🤣 

I'll have a look to see if that's do-able at some point